src/utils/Fetcher.ts

```diff
@@ -55,6 +55,17 @@ export class Fetcher {
             }
         }, options.attempts, options.waitTime)
     }
+
+    async queryGraph<T = any>(url: string, query: string, variables: Record<string, any>): Promise<T> {
```

Comment:
Could we add a `options?: RequestOptions` parameter, like the other public methods